scripts/create_res_drawable.R

```diff
@@ -0,0 +1,471 @@
+## From the Android docs
+## https://developer.android.com/guide/topics/resources/drawable-resource
+## "A drawable resource is a general concept for a graphic that can be drawn to the screen"
+## This script creates the graphics that are shown underneath certain puzzle descriptions
+## For example, app/src/main/res/drawable-hdpi/backgammon_small.png
+## is shown in a puzzle about the probability of certain jumps in the game of backgammon
+
+library(ggplot2)
+library(grid)
+
+source("utils.R")
+
+set_ggplot_theme()
+theme_update(panel.grid.major=element_blank())
+theme_update(panel.grid.minor=element_blank())
+theme_update(axis.text=element_text(size=rel(1.6)))
+theme_update(axis.text.x=element_text(size=rel(1.6)))
+theme_update(axis.text.y=element_text(size=rel(1.6)))
+df <- data.frame(x=c(0, 0, 1, 1),
+                 y=c(0, -1, 0, -1),
+                 label=c("?", "?", "?", "0.6*2/3"))
+colors <- c("#a6611a", "#018571")
+jitter <- 0.05
+p <- (ggplot(df, aes(x=x, y=y, label=label)) +
+      geom_rect(xmin=-0.5+jitter, xmax=1.5+jitter, ymin=-1.5+jitter, ymax=-0.5+jitter,
+                fill="white", color=colors[1], size=3.0, alpha=0) +  # Horizontal
+      annotate("text", x=1.8, y=-1+jitter, color=colors[1], label="0.6", size=11) +
+      geom_rect(xmin=0.5, xmax=1.5, ymin=-1.5, ymax=0.5, fill="white", color=colors[2], size=3.0, alpha=0) +  # Vertical
+      annotate("text", x=1, y=0.65, color=colors[2], label="0.6", size=11) +
+      geom_text(size=8) +
+      scale_y_continuous("voter 3", breaks=c(-1, 0), labels=c("C", "T"), lim=c(-2, 1)) +
+      scale_x_continuous("voter 4", breaks=c(0, 1), labels=c("T", "C"), lim=c(-1, 2)))
+p
+ggsave("CT_vote_table.png", p, width=8, height=8)
```

Comment:
TODO Make sure this script still runs

if it does, we need to make sure (A) that it generates _all_ of the .png drawables used by the app, and (B) that they are the same size as those used in the app. I think I may have edited some of them using GIMP to change the image size, but I could do that directly in the script